flake.nix

```diff
@@ -106,6 +106,7 @@
               "-DBUILD_SHARED_LIBS=ON"
               "-DMANIFOLD_PAR=${if parallel then "ON" else "OFF"}"
             ];
+            doCheck = true;
```

Comment:
And any idea why build_nix_tbb is so slow? It looks like it's building all of TBB before it even gets to Manifold...

Reply:
I added the latest tbb version for emscripten, forgot why I also use it for other native builds... fixed.